sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -51,6 +51,11 @@
   font: unset !important;
 }
 
+// Temporary fix for Vuetify beta that does not display the radio button properly.
+.v-checkbox .mdi:before {
+  font: unset !important;
```

Comment:
for my understanding what is the issue here?

Reply:
The component was not show if we use it directly, so overriding the existing style to display it. It is similar to the v-radio in the above lines.